frontend/src/concepts/pipelines/content/pipelinesDetails/pipelineRun/runLogs/LogsTab.tsx

```diff
@@ -154,72 +148,62 @@ const LogsTabForPodName: React.FC<{ podName: string }> = ({ podName }) => {
               onClick={() => setIsPaused(!isPaused)}
               isDisabled={!!error}
             >
-              {error ? (
-                'Error loading logs'
-              ) : !logsLoaded ? (
-                <>
-                  <Spinner size="sm" /> {isSmallScreen() ? 'Loading' : 'Loading log'}
-                </>
-              ) : isPaused ? (
-                <>
-                  <PlayIcon /> {isSmallScreen() ? 'Resume' : 'Resume refreshing'}
-                </>
-              ) : (
-                <>
-                  <PauseIcon /> {isSmallScreen() ? 'Pause' : 'Pause refreshing'}
-                </>
-              )}
+              {!error ? (
+                !logsLoaded ? (
+                  <>
+                    <Spinner size="sm" /> {isSmallScreen() ? 'Loading' : 'Loading log'}
+                  </>
+                ) : isPaused ? (
+                  <>
+                    <PlayIcon /> {isSmallScreen() ? 'Resume' : 'Resume refreshing'}
+                  </>
+                ) : (
+                  <>
+                    <PauseIcon /> {isSmallScreen() ? 'Pause' : 'Pause refreshing'}
+                  </>
+                )
+              ) : null}
             </Button>
           </SplitItem>
           <SplitItem isFilled style={{ textAlign: 'right' }}>
+            <Tooltip position="top" content={<div>View raw logs</div>}>
+              <Button
+                onClick={() => {
+                  const newTab = window.open();
+                  newTab?.document.write(`<pre>${logs}</pre>`);
+                }}
+                variant="link"
+                aria-label="View raw logs"
+                icon={<OutlinedWindowRestoreIcon />}
+                isDisabled={logs === null || logs === ''}
+              >
+                {!isSmallScreen() ? 'Raw' : null}
+              </Button>
+            </Tooltip>
```

Comment:
@yih-wang The view raw logs functionality is added here. It is reduced to an icon on a small screen and is disabled when there are no logs. Here is a GIF for your reference.

https://github.com/opendatahub-io/odh-dashboard/assets/22885912/e0db2585-1147-444f-bb7a-76444b91357c

Screenshots:

Enabled state: 

<img width="660" alt="Screenshot 2023-11-30 at 7 36 07 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/22885912/25734410-f015-46b3-833e-a4e903de83fb">

Disabled: 
<img width="570" alt="Screenshot 2023-11-30 at 7 50 23 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/22885912/4dbfca87-20f2-488f-9fcd-4c72b4f40bbd">

Small screen:
<img width="390" alt="Screenshot 2023-11-30 at 8 06 31 PM" src="https://github.com/opendatahub-io/odh-dashboard/assets/22885912/3e886888-5608-472f-bd73-9dd30c2c26aa">